controllers/api.controller.js

```diff
@@ -104,3 +106,19 @@ exports.patchVoteByArticle = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.deleteComment = async (req, res, next) => {
+  try {
+    const { comment_id: commentId } = req.params;
+    if (isNaN(commentId)) {
+      return res.status(400).send({ msg: "Invalid comment_id" });
+    }
+    if (!(await fetchCommentsByComId(commentId))) {
```

Comment:
You have chosen to check if the comment exists but this can also be inferred from the db returning empty rows when you run the `DELETE` statement in the model